containers/cpp-mariadb/.devcontainer/install-mariadb.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -e
+
+. /etc/os-release
+
+MARIADB_REPO_SETUP=mariadb_repo_setup
+TMP_DIR=$(mktemp -d -t maria-XXXXXXXXXX)
+MARIADB_CONNECTOR=""
+
+function cleanup {
```

Comment:
you don't need to say `function` to define the func name in this case, you can just do:

`cleanup() { ... }`